packages/react-core/src/components/JumpLinks/JumpLinks.tsx

```diff
@@ -46,6 +46,8 @@ export interface JumpLinksProps extends Omit<React.HTMLProps<HTMLElement>, 'labe
   toggleAriaLabel?: string;
   /** Class for nav */
   className?: string;
+  /** Whether the navigation history should be replaced rather than pushed */
```

Comment:
```suggestion
  /** Whether the current entry in the navigation history should be replaced when a JumpLinksItem is clicked. By default a new entry will be pushed to the navigation history. */
```

Maybe something like this for the description? Just to make it clear that only the current history item would be replaced rather than the entire history being replaced.